activestorage/lib/active_storage/service/registry.rb

```diff
@@ -22,6 +22,10 @@ def fetch(name)
       end
     end
 
+    def names
```

Comment:
This method will need a documentation comment if we intend to make it public

Reply:
Yeah I wasn't sure about this. The class is nodoc but `ActiveStorage::Blob.services.fetch` is already referenced in the guides: https://guides.rubyonrails.org/active_storage_overview.html#cleaning-up-fixtures